docs/units/equivalencies.rst

```diff
@@ -90,6 +91,27 @@ The example with complex numbers is also one may well be doing a fair
 number of similar calculations.  For such situations, there is the
 option to :ref:`set default equivalencies <equivalency-context>`.
 
+This equivalency may result in some unanticipated behavior, such as
+when converting from an angular velocity :math:`\omega` in radians per
+second to the corresponding frequency :math:`f` in hertz, where the
+correct relation is :math:`f=\omega/2\pi`. The
```

Comment:
I find "correct" here a bit misleading; what `dimensionless_angles` does is in fact correct; the problem occurs because `Hz` is taken as a shortcut for `cycles/second`. Probably easiest if I edit directly...